### ISSUE TITLE
Heuristic for calling macros in REPL

### ISSUE THREAD

**Issue by eriksvedang** [carp-lang/Carp#1023]
Right now the just get expanded, which is rarely what you want.

**Comment by scolsen** [comment]
Right, we also have `expand` for getting the expanded form. Probably we can just default to expansion + evaluation and rely on `expand` for the current behavior?

**Comment by eriksvedang** [comment]
That sounds very reasonable (in theory...) let's try it!
